armi/reactor/tests/test_components.py

```diff
@@ -309,10 +309,12 @@ def test_densityConsistent(self):
 
```

Comment:
On line 306 there is a conditional that if `c` is a `Component` or `DerivedShape` that this test is skipped. Aren't all instances of `c` a `Component`? If so, none of the tests in `test_densityConsistent` are ever actually hit... Unless I'm misinterpreting?

Reply:
Yeah, this PR isn't supposed to do anything but change variable names.

But I tested it out and that test wasn't being run.  And when I tried to full reinstate it, it's clear it is used for LOTS of different types of `Component` subclasses that it couldn't possibly pass for (for instance, some components don't have a known volume).

So I deleted the broken half of this test, and revived the rest of it the best I could.

Done